File: spring_java_orm_sample/src/main/java/com/example/demo/jooq/entity/PurchaseInvoiceRow.java
package com.example.demo.jooq.entity;

import lombok.Value;

import java.time.LocalDate;
import java.time.LocalDateTime;

@Value
public class PurchaseInvoiceRow {
    Long id;
    Integer purchaseInvoiceType;
    Long returnPurchaseInvoiceId;
    LocalDate purchaseInvoiceDate;
    Long supplierId;
    Long receivingStoreId;
    Long purchaseInvoiceAmount;
    LocalDateTime updateAt;
    Long version;
}

File: codex_poting/kotlin/src/main/kotlin/com/example/demo/jooq/entity/PurchaseInvoiceRow.kt
package com.example.demo.jooq.entity

import java.time.LocalDate
import java.time.LocalDateTime

data class PurchaseInvoiceRow(
    val id: Long?,
    val purchaseInvoiceType: Int?,
    val returnPurchaseInvoiceId: Long?,
    val purchaseInvoiceDate: LocalDate?,
    val supplierId: Long?,
    val receivingStoreId: Long?,
    val purchaseInvoiceAmount: Long?,
    val updateAt: LocalDateTime?,
    val version: Long?
)
